astropy/modeling/tests/test_quantities_parameters.py

```diff
@@ -341,3 +342,19 @@ def test_magunit_parameter():
     model = Const1D(c)
 
     assert model(-23.0 * unit) == c
+
+
+def test_log_getter():
+    """Regression test for issue #14511"""
+
+    class CustomBlackBody(BlackBody):
+        scale = Parameter(
+            "scale",
+            default=0,
+            bounds=(-1000, 1000),
```

Comment:
```suggestion
            default=1,
            bounds=(0, None),
```
As noted in the issue, I wrote my example code with values for `default` and `bounds` assuming the `getter`/`setter` reparameterizations were applied to them under the hood. Since they aren't, this suggestion uses more sensible values.

Reply:
In reality, these settings make little difference as the bug is exercised via the how parameters and the setting of parameter values is exercised during model initialization. The test simply evolved out of the initial bug report.